browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -165,17 +165,32 @@ void HttpRequestMonitor::Clear() {
 
 EphemeralStorageBrowserTest::EphemeralStorageBrowserTest()
     : https_server_(net::EmbeddedTestServer::TYPE_HTTPS) {
-  SetUpHttpsServer();
 }
 
 EphemeralStorageBrowserTest::~EphemeralStorageBrowserTest() = default;
 
+void EphemeralStorageBrowserTest::SetUp() {
+  ASSERT_TRUE(https_server_.InitializeAndListen());
+  InProcessBrowserTest::SetUp();
+}
+
 void EphemeralStorageBrowserTest::SetUpOnMainThread() {
-  InProcessBrowserTest::SetUpOnMainThread();
```

Comment:
why this one disappeared

Reply:
the base class method does nothing so most tests don't call it. I can add it back though.